python/mxnet/numpy/linalg.py

```diff
@@ -839,7 +889,8 @@ def eigvals(a):
     return _mx_nd_np.linalg.eigvals(a)
 
 
-def eigvalsh(a, UPLO='L'):
+@wrap_data_api_linalg_func
+def eigvalsh(a, upper=False):
```

Comment:
Should Cholesky take a special param `upper=False`? [link](https://github.com/apache/incubator-mxnet/pull/20633/files#diff-ea841d142011d5eecb5db0d91a54f8e60db58da7ae51a02145ecdb4ea50ffb9eR394). According to [link](https://data-apis.org/array-api/latest/extensions/linear_algebra_functions.html#linalg-cholesky-x-upper-false) , a new one op should have additional param.